modules/core/src/main/java/org/apache/ignite/internal/processors/query/RunningQueryManager.java

```diff
@@ -102,20 +139,41 @@ public Long register(String qry, GridCacheQueryType qryType, String schemaName,
      * Unregister running query.
      *
      * @param qryId Query id.
-     * @param failed {@code true} In case query was failed.
+     * @param failReason exception that
```

Comment:
let's add to javaDoc information about null value for the parameter.

Reply:
done.